test/Microsoft.Dnx.TestHost.Tests/ProjectTest.cs

```diff
@@ -0,0 +1,151 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using Microsoft.Dnx.Runtime;
+using Microsoft.Dnx.Runtime.Infrastructure;
+using Xunit;
+
+namespace Microsoft.Dnx.TestHost
+{
+    public class ProjectTest
+    {
+        private const string projectName = "Sample.Tests";
+        private readonly string _testProjectPath;
+
+        public ProjectTest()
+        {
+            var services = CallContextServiceLocator.Locator.ServiceProvider;
+
+            var libraryManager = (ILibraryManager)services.GetService(typeof(ILibraryManager));
+            _testProjectPath = Path.GetDirectoryName(libraryManager.GetLibrary(projectName).Path);
+        }
+
+        [Fact]
+        public void GetsProjectInfo_FromProjectDirectoryPath()
+        {
+            // Arrange
+            var projectDirPath = _testProjectPath;
```

Comment:
What's the value of this? Add a comment with an example so we can see what's being tested